packages/vite-plugin-svelte/src/utils/sourcemaps.ts

```diff
@@ -1,20 +1,60 @@
 import path from 'path';
+const IS_WINDOWS = process.platform === 'win32';
+interface SourceMapFileRefs {
+	file?: string;
+	sources?: string[];
+	sourceRoot?: string;
+}
+/**
+ * convert absolute paths in sourcemap file refs to their relative equivalents to avoid leaking fs info
+ *
+ *
+ */
+export function mapToRelative(map: SourceMapFileRefs | undefined, filename: string) {
+	if (!map) {
+		return;
+	}
+	const dirname = path.dirname(filename);
+	const toRelative = (s: string) => {
+		//remove leading file:// and extra / from file:///C://path on windows
+		let sourcePath = s.startsWith('file:///') ? s.slice(IS_WINDOWS ? 8 : 7) : s;
```

Comment:
Could we use `fileUrlToPath` to handle removing `file://`?

Reply:
maybe, but is that always correct? On windows fileUrlToPath would convert to backslashes again and it's probably slower too.

Reply:
Checking Vite's codebase, looks like it always normalizes the path so maybe this manual handling is fine then.